src/test/scala/org/broadinstitute/dsde/firecloud/webservice/TrialApiServiceSpec.scala

```diff
@@ -327,6 +383,8 @@ final class TrialApiServiceSpec extends BaseServiceSpec with UserApiService with
     override def isGroupMember(userInfo: UserInfo, groupName: String): Future[Boolean] = {
       userInfo.id match {
         case "failme" => Future.failed(new Exception("intentional exception for unit tests"))
+        case "campaignManager" => Future.successful(true)
```

Comment:
Note that if you use the `manager` val defined in the companion object, you should be able to remove the case above since `"manager"` is defined as one of the `"trial_managers"` above. In my previous PR, I meant to turn 

```scala 
"trial_managers" -> Seq("manager")
```
into 
```scala
"trial_managers" -> Seq(manager)
```
but forgot. It will be in my next PR and you're welcome to do that here, too.

Reply:
👍 Got both in, thank you!